Utilities/QC/QCMerger/src/MergerDevice.cxx

```diff
@@ -36,8 +36,8 @@ MergerDevice::MergerDevice(unique_ptr<Merger> merger, string mergerId, int numIo
   : mMerger(move(merger)), ddsCustomCmd(new CCustomCmd(mService))
 {
   this->SetTransport("zeromq");
-  this->SetProperty(Id, mergerId);
-  this->SetProperty(NumIoThreads, numIoThreads);
+  this->SetId(mergerId);
+  this->SetNumIoThreads(numIoThreads);
```

Comment:
It is a remnant. We will have to rework this prototype later this year. You can deprecate it.